schema/deploy/warehouse/sample.sql

```diff
@@ -0,0 +1,34 @@
+-- Deploy seattleflu/schema:warehouse/sample to pg
+-- requires: warehouse/encounter
+
+begin;
+
+set search_path to warehouse;
+
+create table sample (
+    sample_id integer primary key generated by default as identity,
+    identifier text unique not null,
+    encounter_id integer, -- references encounter (encounter_id), not null,
+    details jsonb
+);
+
+-- We removed the encounter_id reference to the encounter table because this relationship
+--   does not exist yet in the database. We will add the relationship back in once
+--   Thomas has finished with the barcode ingestion part of the pipeline.
+--   At that time, we can also decide whether or not to add a `not null` constraint to 
+--   encounter_id. 
```

Comment:
I have the barcode ingestion process designed and worked out in my head, but have been at a loss for time to actually write it. It's my top priority for next week. :-) The encounter id will be nullable due to the possibility of receiving results for which we don't yet (or never will) have encounters. I can walk you both through my plans for barcode ingestion if you'd like, and how it affects your presence/absence work.